Orchestrator-CredentialStorePlugins-Samples.sln

```diff
@@ -16,6 +16,14 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CyberArkCCP.SecureStore", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SecureStore.HashicorpVault", "src\SecureStore.HashicorpVault\SecureStore.HashicorpVault.csproj", "{BB4B920F-0F66-4BF2-87E9-8E358A876D62}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SecureStore.BeyondTrust", "SecureStore.BeyondTrust", "{736ED120-C952-4291-BA94-0D35479C0B81}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UiPath.Orchestrator.BeyondTrust", "src\SecureStore.BeyondTrust\UiPath.Orchestrator.BeyondTrust\UiPath.Orchestrator.BeyondTrust.csproj", "{90FBAD13-FA5D-472A-B5A5-1A920819A547}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UiPath.Orchestrator.BeyondTrust-DynamicSystem-ReadOnly", "src\SecureStore.BeyondTrust\UiPath.Orchestrator.BeyondTrust-DynamicSystem-ReadOnly\UiPath.Orchestrator.BeyondTrust-DynamicSystem-ReadOnly.csproj", "{B3955F49-4B07-4CEE-9544-8D8369669124}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UiPath.Orchestrator.BeyondTrust-SingleSystem-ReadOnly", "src\SecureStore.BeyondTrust\UiPath.Orchestrator.BeyondTrust-SingleSystem-ReadOnly\UiPath.Orchestrator.BeyondTrust-SingleSystem-ReadOnly.csproj", "{C1AAB92C-66E7-4C5E-8A82-10B3812C3E7E}"
```

Comment:
I don't think we need so many projects. You can have multiple credential store classes in a single dll, so why not have just one?

Reply:
Merged all into a single project